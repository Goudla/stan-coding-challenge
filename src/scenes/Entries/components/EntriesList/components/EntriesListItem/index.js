import React from 'react';

// Components
import EntryImagesPosterArtUrl from '../../../../../../components/EntryImagesPosterArtUrl';
import Title from '../../../../../../components/Title';

type Props = {
  entryTitle: string
};

class EntriesListItem extends React.PureComponent<Props> {
  render() {
    const { entryTitle } = this.props;
    return (
      <div>
        <div>
          <EntryImagesPosterArtUrl entryTitle={entryTitle} />
        </div>
        <div>
          <Title>{entryTitle}</Title>
        </div>
      </div>
    );
  }
}

export default EntriesListItem;
