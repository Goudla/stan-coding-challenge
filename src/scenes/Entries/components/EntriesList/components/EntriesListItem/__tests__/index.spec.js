import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import EntriesListItem from '../';

describe('<EntriesListItem />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<EntriesListItem entryTitle="test" />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
