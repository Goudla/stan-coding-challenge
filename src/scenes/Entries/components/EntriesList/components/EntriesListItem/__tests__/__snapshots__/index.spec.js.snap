// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EntriesListItem /> Should render 1`] = `
<div>
  <div>
    <Connect(EntryImagesPosterArtUrl)
      entryTitle="test"
    />
  </div>
  <div>
    <Title>
      test
    </Title>
  </div>
</div>
`;
